### PR TITLE
Add V1 Meeting documentation for delete

### PR DESCRIPTION
**Pull request by Jdww**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -879,6 +879,19 @@ Delete a file
...
+## Meetings [/meetings]
+
+### /deleteMeeting.php [POST /deleteMeeting.php]
+
+Delete a meeting from Teamleader
+
++ Request (application/json)
+
+    + Attributes (object)
+        + meeting_id: `1` (number, required) - The ID of the meeting you wish to delete
```
Comment (stivni) [inline apiary.apib]: Q: are we going to allow old way of authentication on this endpoint too?
where do we reflect this kind of decision?

Comment (Jdww) [inline apiary.apib]: Not sure if I understand the question. This endpoint goes to api/deleteMeeting.php which uses the old api key/secret when running through setup.php. Inside deleteMeeting.php we direct it to the DeleteMeetingRequestHandler which is also called using api/meetings.delete which uses api/index.php for authentication. So we support the old way of authentication in deleteMeeting.php and the new way in meeting.delete.

File: apiary.apib [hunk 2]
```diff
@@ -879,6 +879,19 @@ Delete a file
             "status": "success"
         }
 
+## Meetings [/meetings]
+
+### /deleteMeeting.php [POST /deleteMeeting.php]
+
+Delete a meeting from Teamleader
+
++ Request (application/json)
```
Comment (jenssegers) [inline apiary.apib]: `multipart/form-data` here or json?

Comment (Jdww) [inline apiary.apib]: I changed it in https://github.com/teamleadercrm/api/pull/112 to application/x-www-form-urlencoded

Comment (jenssegers) [inline apiary.apib]: I'm not sure if `application/x-www-form-urlencoded` actually works.

File: apiary.apib (hunk 1 above)
Comment (jenssegers) [inline apiary.apib]: For products this was also `product_id`:

```
product_id: id: id of the product
```

File: apiary.apib [hunk 3]
```diff
@@ -879,6 +879,19 @@ Delete a file
...
+
+    + Attributes (object)
+        + meeting_id: `1` (number, required) - The ID of the meeting you wish to delete
+
++ Response 204
```
Comment (jenssegers) [inline apiary.apib]: Should we return 204 for v1 endpoints? Most return a json status message right?

Comment (Jdww) [inline apiary.apib]: Most of them return text or JSON but not sure if this is an issue as it's a new endpoint and it's not consistent at all atm. We are using the same replies as meetings.delete in v2.

**Pull request merged**